README.md

```diff
@@ -6,31 +6,47 @@ Hound is an extremely fast source code search engine. The core is based on this
 [Regular Expression Matching with a Trigram Index](http://swtch.com/~rsc/regexp/regexp4.html). Hound itself is a static
 [React](http://facebook.github.io/react/) frontend that talks to a [Go](http://golang.org/) backend. The backend keeps an up-to-date index for each repository and answers searches through a minimal API. Here it is in action:
 
-![Hound Screen Capture](screen_capture.gif)
+![Hound Screen Capture](imgs/screen_capture.gif)
 
 ## Quick Start Guide
 
 ### Using Go Tools
 
+0. [Install the Go tools](https://golang.org/doc/install) if you don't have it. 
```

Comment:
We could probably make this a bit terser:
> 0. [Install Go](https://golang.org/doc/install) if you don't have it already. Hound requires version 1.4 or later. You might also want to define a [`GOPATH` environment variable](https://github.com/golang/go/wiki/GOPATH)) (it defaults to `$HOME/go` if you don't explicitly have one set). If everything is installed properly, `go version` will print out the installed version of go.

Reply:
Thanks for your suggestion. Updated.